### ISSUE TITLE
Breakfast: frame accuracy: 0.375708

### ISSUE THREAD

**Issue by jszgz** [alexanderrichard/NeuralNetwork-Viterbi#9]
> Evaluate 252 video files...
> frame accuracy: 0.375708

Hello, I downloaded the dataset you provided and ran your code, but only get  0.375708 on Breakfast dataset. By the way, is it segmentation acc or alignment acc?

**Comment by alexanderrichard** [comment]
Hi!

It's segmentation acc but it should still be above 40%.

The numbers in the paper have been computed with an in-house C++/Cuda based neural network framework that is no longer maintained. I therefore provided the re-implementation in pytorch. I was able to recreate the original results from the paper and so were a bunch of others using this framework.

However, you are not the first facing these problems. I assume due to differences in the implementation, pytorch undergoes more fluctuation among different runs. The original implementation used a different weight/bias initialization: weights are uniformly sampled from [-0.1, 0.1] and biases are uniformly sampled from [-1, 1]. I suggest you try this weight/bias init with pytorch. Let me know if it solves the problem!
